app/views/hyrax/base/_guts4form.html.erb

```diff
@@ -0,0 +1,80 @@
+<!-- OVERRIDE: Add in new tab -->
+<%# we will yield to content_for for each tab, e.g. :files_tab %>
+<%# Not passing tabs local param to this partial is deprecated and the tabs param will be required in Hyrax 3.0 %>
+<% unless defined?(tabs) %>
+  <% Deprecation.warn(self, "Passing the tabs local param to the _guts4form partial will be required in Hyrax 3.0.  " \
+                            "Consider removing overriding view partials and customizing the tab list " \
+                            "by overriding the new form_tabs_for(form:) helper.") %>
+  <% tabs = form_tabs_for(form: f.object) # default tab order %>
+<% end %>
+<div class="row">
+  <div class="col-xs-12 col-sm-8">
+    <div class="panel panel-default tabs" role="main">
+      <!-- Nav tabs -->
+      <ul class="nav nav-tabs" role="tablist">
+        <% tabs.each_with_index do | tab, i | %>
+          <% if i == 0 %>
+            <li role="presentation" class="active">
+          <% else %>
+            <li role="presentation">
+          <% end %>
+              <a href="#<%= tab %>" aria-controls="<%= tab %>" role="tab" data-toggle="tab">
+                <%= t("hyrax.works.form.tab.#{tab}") %>
+              </a>
+            </li>
+        <% end %>
+
+        <% # NEW TAB: Add in a tab for OCR text display for admin %>
+        <% if f.object.model.persisted? && current_user.admin? && !f.object.model.file_sets.blank? %>
```

Comment:
Can you remove this override and use a system like this: https://github.com/osulp/Scholars-Archive/blob/main/app/helpers/hyrax/doi/work_form_helper.rb#L7-L12

You can add your method around here: https://github.com/osulp/Scholars-Archive/blob/main/app/helpers/hyrax_helper.rb#L24

As long as the value you insert is `ocr` your other view should just work

Reply:
Got it, I will try to work on changing the view. Also the code system you reference, I see that it do `super.insert(1, doi)`. And was wondering if that `1` is the index or insert 1 new value in?

Reply:
Correct, it's the index to insert into. So you can put it in a specific place or if you just want to put it on the end, use the shovel `<<`